endpoints/_includes/functions.php

```diff
@@ -736,20 +736,18 @@ function helplineSearch($latitude, $longitude)
 
 function isBMLTServerOwned($latitude, $longitude)
 {
-    $bmlt_search_endpoint = sprintf('%s/client_interface/json/?switcher=GetSearchResults&data_field_key=root_server_uri&sort_results_by_distance=1&long_val=%s&lat_val=%s&geo_width=%s',
+    $bmlt_search_endpoint = sprintf('%s/client_interface/json/?switcher=GetSearchResults&data_field_key=root_server_uri&sort_results_by_distance=1&lat_val=%s&long_val=%s&geo_width=%s',
         setting('tomato_url'), $latitude, $longitude, setting('helpline_search_radius'));
     $search_results = json_decode(get($bmlt_search_endpoint));
     $root_server_uri_from_first_result = $search_results[0]->root_server_uri;
-    return strpos(getAdminBMLTRootServer(), $root_server_uri_from_first_result) == 0;
+    return ($root_server_uri_from_first_result == getAdminBMLTRootServer().'/');
```

Comment:
This result will fail if the user specified an ending `/` in the configuration.

Reply:
btw, what was not working with what you had?

Reply:
the previous wasn’t mine - it wasn’t correctly returning true/false against is the server owned

Reply:
Unless I'm missing something, this line was authored by you.

`return strpos(getAdminBMLTRootServer(), $root_server_uri_from_first_result) == 0;`

Reply:
`strpos` is a weird function, it returns a position if true.

Reply:
There is a custom function you could use called `str_exists()`

Reply:
Actually I did not - it may show I did from the awkward rebasing I did - but strpos wasn’t me. The str_exists may work better though since it may have just been the / appended at the end of how tomato returns the urls not matching the bmlt server settings in config that do not have / at the end.

With the bug fix you had me do yesterday, everything is working now though

Reply:
ok I'm going to close this pull request out and make the cleanup changes.